schematics/nav/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.__styleext__

```diff
@@ -0,0 +1,8 @@
+mat-drawer-container {
+  height: 100%;
+}
+
+mat-drawer {
+  width: 200px;
+  box-shadow: 3px 0 6px rgba(0,0,0,.24);
```

Comment:
Should this be done with one of the elevation classes instead?

Reply:
I thought about using the elevation helper but that would only work if you are using SCSS. We have to assume CSS/inline css/etc.